### PR TITLE
feat: deposit fix

### PR DESCRIPTION
**Pull request by Wolmin**
(no clickup ticket since it was made on the fly)
This is a PR that introduces:
- `--checkpoint-sync` bool flag - enables checkpoint sync based on your selected network
- hardcoded `--contract-deployment-block=0` into prysm's beacon client
- Geth downgraded to 1.11.6

### CODE REVIEW

**Review by mxmar — APPROVED** [review]
LGTM. Did we test it already? @Wolmin @fhildeb

**Comment by mxmar** [comment]
Tested on `Ubuntu 22.04 LTS amd64` ->  `lukso start --checkpoint-sync`:

```
ps aux | grep prysm
prysm --accept-terms-of-use --config-file=./configs/mainnet/prysm/prysm.yaml --contract-deployment-block=0 --genesis-state=./configs/mainnet/shared/genesis.ssz --chain-config-file=./configs/mainnet/shared/config.yaml --checkpoint-sync-url=https://checkpoints.mainnet.lukso.network/
```

**Comment by fhildeb** [comment]
Still WIP: testing on Debian and ARM now

File: commands/start.go [hunk 1]
```diff
@@ -54,6 +54,17 @@ func StartClients(ctx *cli.Context) (err error) {
 		return utils.Exit(fmt.Sprintf("❌  There was an error while preparing %s flags: %v", consensusClient.Name(), err), 1)
 	}
 
+	if ctx.Bool(flags.CheckpointSyncFlag) {
+		log.Info("⚙️   Checkpoint sync feature enabled")
+
+		checkpointURL := "https://checkpoints.mainnet.lukso.network/"
```
Comment (frozeman) [inline commands/start.go]: what if its devote, and a user provides this flag, will it error to the user?

Comment (frozeman) [reply commands/start.go]: or will it try to use the mainnet, which would not work.

Comment (Wolmin) [reply commands/start.go]: > what if its devote, and a user provides this flag, will it error to the user?

Currently devnet is disabled, so it wont get to this point

Comment (mxmar) [reply commands/start.go]: @Wolmin We will need to secure it.

Comment (frozeman) [reply commands/start.go]: on devote or any other future network please just log with (no error or exit, just skip the flag) with:
`"This network doesn't have a checkpoint sync setup, starting without checkpoint sync..."`

Comment (frozeman) [reply commands/start.go]: please also make the URLs global vars,

Comment (mxmar) [reply commands/start.go]: Solved. Secured and move checkpoints URLs to config vars.

**Review by frozeman — APPROVED** [review]

**Pull request merged**